crates/wysiwyg/src/dom/nodes/mention_node.rs

```diff
@@ -96,9 +123,19 @@ where
         1
     }
 
-    pub fn kind(&self) -> &MentionNodeKind<S> {
+    pub fn kind(&self) -> &MentionNodeKind {
         &self.kind
     }
+
+    /// Util function to check if the display text is that of an at-room mention
+    pub fn is_at_room_display_text(text: &S) -> bool {
+        text == &S::from("@room")
+    }
+
+    /// Util function to get the display text for an at-room mention
+    pub fn get_at_room_display_text() -> S {
+        S::from("@room")
+    }
```

Comment:
Could these utils and the `"@room"` constant be part of `matrix_mentions` crate?